packages/gatsby-source-contentful/README.md

```diff
@@ -392,5 +392,30 @@ documentToReactComponents(node.bodyRichText.json, options)
 
 Check out the examples at [@contentful/rich-text-react-renderer](https://github.com/contentful/rich-text/tree/master/packages/rich-text-react-renderer).
 
+## Sourcing From Multiple Contentful Environments
```

Comment:
One suggestion Here: `Environments` might be confusing here since there is already the concept of [Environment](https://www.contentful.com/faq/environments/) already.
Maybe change this to 
```suggestion
## Sourcing From Multiple Contentful Spaces
```

Reply:
I wasn't sure, as this does allow you to import content from both multiple environments or multiple spaces. It's entirely possible both the environments they import could be from the same space. Thoughts?